index.html

```diff
@@ -42,7 +42,7 @@
           <a class="desktop-link" href="#">Contact</a>
```

Comment:
- [ ] You are using a `<p>` tag for the first project title and the `<h2>` tag for the next projects. I would suggest updating this heading for a better structure and accessibility :smiley:
![image](https://user-images.githubusercontent.com/81522435/141004772-aed64712-2060-494a-ae19-fe3e115d9d49.png)
- [ ] Also, you're using the `<h2>` tag for the section title "My Recent Works", and I think it could be a good idea to use the `<h3>` tags for the project since these are below (or are less important semantically speaking) than the section title :pushpin: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/141005049-7fd2d754-e96d-4f2f-b33e-d7f915d3aa76.png)

- [ ] There is an issue with the design of the buttons because these are not passing the contrast check. I would suggest fixing this contrast issue for better accessibility :wink:
**You can see the [contrast test results here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2021-11-09-at-21-10-27-367?failing)**
![image](https://user-images.githubusercontent.com/81522435/141005399-8f010f90-ac29-4ff9-a952-836a0f4a5c41.png)

Reply:
I don't know why the hover did not work. Guess it was the transition, but the error kept coming back. Change colors to a darker theme and now it's fine.